apps/web/src/style.css

```diff
@@ -1,30 +1,30 @@
 * {
-    box-sizing: border-box;
-    font-family: Helvetica, Arial, sans-serif;
+  box-sizing: border-box;
+  font-family: Helvetica, Arial, sans-serif;
 }
 
 body {
-    margin: 0;
-    padding: 0;
-    overflow: hidden;
-    color: #fff;
-    background-color: rgb(36, 36, 40);
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  color: #fff;
+  background-color: rgb(36, 36, 40);
 }
 
 main {
-    padding: 32px;
-    display: grid;
-    width: 100vw;
-    height: 100vh;
-    justify-content: center;
-    align-items: center;
-    overflow: auto;
+  padding: 32px;
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  overflow: auto;
 }
 
 canvas {
-    border: 1px solid rgb(25, 25, 28);
-    background-color: rgb(45, 45, 50);
-    background-image: url("data:image/svg+xml,%3Csvg opacity='0.2' fill='black' xmlns='http://www.w3.org/2000/svg' width='32' height='32' viewBox='0 0 32 32'%3E%3Cpolygon fill-rule='evenodd' points='0,0 16,0 16,32 32,32 32,16 0,16' /%3E%3C/svg%3E");
-    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
-    cursor: none;
+  border: 1px solid rgb(25, 25, 28);
+  background-color: rgb(45, 45, 50);
+  background-image: url("data:image/svg+xml,%3Csvg opacity='0.2' fill='black' xmlns='http://www.w3.org/2000/svg' width='32' height='32' viewBox='0 0 32 32'%3E%3Cpolygon fill-rule='evenodd' points='0,0 16,0 16,32 32,32 32,16 0,16' /%3E%3C/svg%3E");
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+  cursor: crosshair;
```

Comment:
I'll do a better cursor thing later - not sure if I liked it better as none or this for now tbh